src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
That's true, but mitigable by two means: resetting style & innerHTML before use (which we actually do, but can't enforce), and using distinct variable names. At any rate, it's off topic for this discussion so I'll either forget about it or open a PR eventually.